graphql-api/src/queries/variant-datasets/shared/flags.ts

```diff
@@ -126,3 +126,22 @@ export const getLofteeFlagsForContext = (context: any) => {
       throw Error(`Invalid context for getFlags: ${context.type}`)
   }
 }
+
+export const getFlagsForContext = (context: any, variant: any) => {
+  const baseVariantFlags = getLofteeFlagsForContext(context)(variant)
+  const baseExomeFlags = variant.exome?.flags || []
+  const baseGenomeFlags = variant.genome?.flags || []
+
+  const regionalFlags = ['par', 'segdup', 'lcr'].filter(
```

Comment:
Based on the hail tables help text, it appears as though the par flag might be named "non_par"? 

I did a quick grep and didn't see any place where we rename this flag in our pipeline, so just wanted to check.

Reply:
The v4 data pipeline does address this, https://github.com/broadinstitute/gnomad-browser/blob/b804dbeb4b476ba40ce9354c1cd51be88dc1577c/data-pipeline/src/data_pipeline/datasets/gnomad_v4/gnomad_v4_variants.py#L286, but it would be smart for me to double-check that it's correct.

Reply:
Ah whoops, my bad for not finding that, and nice catch!